frontend/src/features/permits/pages/Amend/types/AmendPermitFormData.ts

```diff
@@ -63,25 +58,20 @@ export const getDefaultFormDataFromPermit = (
   if (!permit) {
     return {
       ...getDefaultValues(defaultPermitType),
-      permitApplicationOrigin: PERMIT_APPLICATION_ORIGINS.ONLINE,
-      permitApprovalSource: PERMIT_APPROVAL_SOURCES.AUTO,
     };
   }
 
+  const {
+    /* eslint-disable-next-line @typescript-eslint/no-unused-vars */
+    issuer,
```

Comment:
Is issuer needed if unused?

Reply:
This removes the "issuer" field from the permit object (rather than using delete permit.issuer) before transforming the permit into an AmendPermitFormData object. The destructuring syntax automatically copies over every field (including issuer which isn't needed), so it extracts and discard it here before destructuring the rest of the permit fields.